src/components/Image/src/Image.vue

```diff
@@ -91,7 +91,7 @@ export default {
 		shape: {
```

Comment:
please add a JSDoc comment above this prop describing what the different values do and it'll automatically get added to the component's styleguide page via the automated API table generation... it's not immediately obvious why `original` exists or what it does...